vm.h

```diff
@@ -31,6 +32,14 @@ typedef struct {
     vm_handler handler;
 } vm_opcode_impl;
 
+typedef struct {
+    enum { SYMBOLIC, NUMERIC } type;
+    char *label;
+    int next_pc;
+} vm_label;
+
+#define LABLE_NOT_FOUND -1
```

Comment:
Using `typedef enum { LABLE_NOT_FOUND = -1, ERR1, ERR2, ERR3, ... } vm_err_t;` is better for diagnosis.